schedule.py

```diff
@@ -112,7 +112,8 @@ def get(self):
       hover_text = logic.GetHoverText(use_full_description, c)
       c['hover_text'] = hover_text
       for daypart in [s['daypart'] for s in c['schedule']]:
-        classes_by_daypart[daypart].append(c)
+        if daypart in classes_by_daypart:
+          classes_by_daypart[daypart].append(c)
```

Comment:
The error check page might be good enough but maybe also if this check ever fails we could consider creating a descriptive error message here and sending it to the page.